api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -54,7 +54,7 @@ public class TestHelpers {
   private TestHelpers() {}
 
   public static final int MAX_FORMAT_VERSION = 3;
-  public static final int[] ALL_VERSIONS = IntStream.rangeClosed(1, MAX_FORMAT_VERSION).toArray();
+  public static final List<Integer> ALL_VERSIONS = List.of(1, 2, 3);
```

Comment:
This should still be using MAX_FORMAT_VERSION and RANGE, the point here is to avoid having to change this and MAX_FORMAT_VERSION

Reply:
We could use another construct as well, but I want to make sure that when we change MAX_FORMAT_VERSION we change ALL_VERSIONS